presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergUtil.java

```diff
@@ -417,6 +417,13 @@ public static void validateTableMode(ConnectorSession session, org.apache.iceber
         }
     }
 
+    public static void verifyTypeSupported(Schema schema)
+    {
+        if (schema.columns().stream().anyMatch(c -> Types.TimestampType.withZone().equals(c.type()))) {
```

Comment:
```suggestion
        if (schema.columns().stream().anyMatch(column -> Types.TimestampType.withZone().equals(column.type()))) {
```
It's better to avoid abbreviated variable names.